crawlerr/server.py

```
@@ -338,7 +380,7 @@ def main():
         logger.info("Server stopped by user")
         sys.exit(0)
     except Exception as e:
-        logger.error(f"Server failed to start: {e}")
+        logger.error("Server failed to start: %s", e)
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)